build.sbt

```diff
@@ -46,6 +46,8 @@ lazy val commonSettings = Seq(
   }
 )
 
+Global / excludeLintKeys ++= Set(Compile / logLevel)
```

Comment:
I believe removing the keys is better rather than excluding them from lint. I don't think there should be any change in behaviour.

Reply:
@vikrantpuppala yeah, that's one of the solutions, but I chose to handle it during linting because I wasn't sure about the intended use of that key. If it's fine to remove, I will update the commit.

Reply:
I played around with this today and noticed no issues with the server and cli output post removal of these settings. I think it's better to have these removed rather than ignoring in lint.